tests/lib/bin/retry

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env any-python
+import argparse
+import subprocess
+import sys
+import time
+
+
+def run_command(commands, max_attempts, sleep_time):
+
+    for _ in range(0, max_attempts):
+        try:
+            subprocess.call(commands)
```

Comment:
This will never fail, as `subprocess.call` just returns the process object.

I'll tweak this a little if you don't mind.

Reply:
go ahead :)